final/server/package.json

```diff
@@ -13,15 +13,15 @@
   "dependencies": {
     "apollo-datasource": "^3.1.0",
     "apollo-datasource-rest": "^3.2.0",
-    "apollo-server": "^3.2.0",
+    "apollo-server": "^3.8.1",
     "aws-sdk": "^2.585.0",
     "dotenv": "^6.2.0",
     "graphql": "^15.5.1",
     "isemail": "^3.1.3",
     "mime": "^2.4.4",
     "nodemon": "^1.19.4",
-    "sequelize": "^5.21.2",
-    "sqlite3": "^5.0.2",
+    "sequelize": "^6.20.1",
+    "sqlite3": "^5.0.8",
```

Comment:
I had to make these changes because the latest version was failing to `npm install` on my MacBook Pro w/ M1 Pro.